pages/manage/productions/index.vue

```diff
@@ -215,53 +224,23 @@
 </script>
 
 <style lang="scss">
-  .productions-container {
-    display: flex;
-    width: 100%;
-    font-weight: 400;
-    margin-bottom: 1rem;
-
-    .table {
-      font-family: 'Open Sans', Helvetica, Arial, sans-serif;
-      font-size: 15px;
-      background-color: none;
-
-      thead {
-        background-color: $white;
-      }
-
-      tbody {
-        margin-top: 0.5rem;
-        background-color: $white;
-      }
-
-      tr {
-        display: inline-flex;
-        width: 100%;
-      }
-
-      th {
-        width: 100%;
-      }
+  .productions-index-page {
```

Comment:
Same suggestion, you can just make this `style` block `scoped` instead of wrapping all the rules inside a single container class.

Reply:
Will be implemented in next PR